pandas/core/arrays/_ranges.py

```diff
@@ -82,17 +84,7 @@ def generate_regular_range(
             "at least 'start' or 'end' should be specified if a 'period' is given."
         )
 
-    with np.errstate(over="raise"):
-        # If the range is sufficiently large, np.arange may overflow
-        #  and incorrectly return an empty array if not caught.
-        try:
-            values = np.arange(b, e, stride, dtype=np.int64)
-        except FloatingPointError:
```

Comment:
Do we have coverage here?

Reply:
I'm assuming there's coverage in the tests added in https://github.com/pandas-dev/pandas/pull/33498 where this code was added